### PR TITLE
update document about TTL

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: YangKeao <yangkeao@chunibyo.icu>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

We have made some modification for the TTL:

1. Remove the system variable `tidb_ttl_job_run_interval`
2. Add a table option `ttl_job_interval`
3. Add some metrics about the distributed execution

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/12318
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-create-table.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ TableOption ::=
 |   'SECONDARY_ENGINE' EqOpt ( 'NULL' | StringName )
 |   'UNION' EqOpt '(' TableNameListOpt ')'
 |   'ENCRYPTION' EqOpt EncryptionOpt
-|    'TTL' EqOpt TimeColumnName '+' 'INTERVAL' Expression TimeUnit (TTLEnable EqOpt ( 'ON' | 'OFF' ))?
+|    'TTL' EqOpt TimeColumnName '+' 'INTERVAL' Expression TimeUnit (TTLEnable EqOpt ( 'ON' | 'OFF' ))? (TTLJobInterval EqOpt stringLit)?
```
Comment (ran-huang) [inline sql-statements/sql-statement-create-table.md:86]: This is [the preview of the diagram](https://test-ebnf-diagram.pingcap-docs-website-scaffold-1hq.pages.dev/zh/tidb/dev/sql-statement-create-table). Please make sure that the rendering is correct. Thanks!

File: system-variables.md [hunk 2]
```diff
@@ -3377,18 +3377,6 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 范围：`[1, 256]`
 - 这个变量用于设置每个 TiDB 节点 TTL 扫描任务的最大并发数。
 
-### `tidb_ttl_job_run_interval` <span class="version-mark">从 v6.5.0 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:3380]: @YangKeao 是否需要补充新增的系统变量？

Comment (YangKeao) [reply system-variables.md:3380]: No. It's replaced by the `ttl_job_interval` attribute, as shown in https://github.com/pingcap/docs-cn/pull/12857/files#diff-cde68168cc89f2560143e5df207cb5cbfc5ab2ad659b9d0d52ff8f2f4469d296R136

File: sql-statements/sql-statement-create-table.md (hunk 1 above)
Comment (YangKeao) [reply sql-statements/sql-statement-create-table.md:86]: Thanks. It's correct :+1: .

File: sql-statements/sql-statement-alter-table.md [hunk 3]
```diff
@@ -53,6 +53,7 @@ AlterTableSpec ::=
         'TTL' EqOpt TimeColumnName '+' 'INTERVAL' Expression TimeUnit (TTLEnable EqOpt ( 'ON' | 'OFF' ))?
         | 'REMOVE' 'TTL'
         | TTLEnable EqOpt ( 'ON' | 'OFF' )
+        | TTLJobInterval EqOpt stringLit
```
[the remark this answers is not in the document]
Comment (YangKeao) [reply sql-statements/sql-statement-alter-table.md:56]: Thanks. It's correct :+1: .

File: time-to-live.md [hunk 4]
```diff
@@ -133,12 +133,14 @@ CREATE TABLE orders (
 
 ## TTL 任务
 
-对于每张设置了 TTL 属性的表，TiDB 内部会定期调度后台任务来清理过期的数据。你可以通过设置全局变量 [`tidb_ttl_job_run_interval`](/system-variables.md#tidb_ttl_job_run_interval-从-v650-版本开始引入) 来自定义任务的执行周期，比如通过下面的语句将后台清理任务设置为每 24 小时执行一次：
+对于每张设置了 TTL 属性的表，TiDB 内部会定期调度后台任务来清理过期的数据。你可以通过给表设置 `ttl_job_interval` 属性来自定义任务的执行周期，比如通过下面的语句将后台清理任务设置为每 24 小时执行一次：
```
Comment (ran-huang) [inline time-to-live.md:136]:
```suggestion
对于每张设置了 TTL 属性的表，TiDB 内部会定期调度后台任务来清理过期的数据。你可以通过给表设置 `TTL_JOB_INTERVAL` 属性来自定义任务的执行周期，比如通过下面的语句将后台清理任务设置为每 24 小时执行一次：
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

File: time-to-live.md (hunk 4 above)
Comment (lcwangchao) [inline time-to-live.md:136]:
```suggestion
对于每张设置了 TTL 属性的表，TiDB 内部会定期调度后台任务来清理过期的数据。你可以通过给表设置 `TTL_JOB_INTERVAL` 属性来自定义任务的执行周期，比如通过下面的语句将后台清理任务设置为每 24 小时执行一次：
```

Comment (lcwangchao) [reply time-to-live.md:136]: 都统一大写吧

**Review by lcwangchao — APPROVED** [review]

**Pull request merged**